tests/plotting/test_actor.py

```diff
@@ -146,6 +156,29 @@ def test_actor_orientation(actor):
     actor.orientation = orientation
     assert np.allclose(actor.orientation, orientation)
 
+    # test rotation order
```

Comment:
Let's divided test code into function so that you can see where errors occur.
```suggestion
    def test_actor_rotation_order(actor):
```